core/src/main/java/org/elasticsearch/action/ReplicationResponse.java

```diff
@@ -280,24 +279,24 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
             private static class Fields {
 
-                private static final XContentBuilderString _INDEX = new XContentBuilderString("_index");
-                private static final XContentBuilderString _SHARD = new XContentBuilderString("_shard");
-                private static final XContentBuilderString _NODE = new XContentBuilderString("_node");
-                private static final XContentBuilderString REASON = new XContentBuilderString("reason");
-                private static final XContentBuilderString STATUS = new XContentBuilderString("status");
-                private static final XContentBuilderString PRIMARY = new XContentBuilderString("primary");
+                private static final String _INDEX = "_index";
```

Comment:
At this point we really should just be passing the string to the method. I expect this change is purely mechanical so that isn't something you want to do here.